qiskit/pulse/library/parametric_pulses.py

```diff
@@ -150,12 +157,13 @@ def __init__(
             sigma: A measure of how wide or narrow the Gaussian peak is; described mathematically
                    in the class docstring.
             name: Display name for this pulse envelope.
+            limit_amplitude: Passed to parent ParametricPulse
```

Comment:
Perhaps a bit more detail could be given about what the arg actually is, instead of just stating that it gets passed to the parent?

Reply:
Thanks @javabster! I've made the change.